### PR TITLE
[HAL-1781] : Enhance check for update feature

### PR DESCRIPTION
**Pull request by fazer1929**
https://issues.redhat.com/browse/HAL-1781

### CODE REVIEW

File: app/src/main/java/org/jboss/hal/client/bootstrap/tasks/CheckForUpdate.java [hunk 1]
```diff
@@ -43,6 +44,16 @@ public CheckForUpdate(Environment environment, EventBus eventBus) {
         this.eventBus = eventBus;
     }
 
+    private String getVersionFromResponse(String responseText) {
+        JSONObject data = new JSONObject(JsonUtils.safeEval(responseText));
+        String version = data.get("tag_name").toString();
+        version = version.substring(1, version.length() - 1);
+        if (version.contains("v")) {
```
Comment (hpehl) [inline app/src/main/java/org/jboss/hal/client/bootstrap/tasks/CheckForUpdate.java:51]: Maybe `version.startsWith("v")` is better here?

Comment (fazer1929) [reply app/src/main/java/org/jboss/hal/client/bootstrap/tasks/CheckForUpdate.java:51]: sure, will change it

**Comment by hpehl** [comment]
Thanks, @fazer1929. I plan another HAL release for Thursday and will review your PR by then.

**Comment by hpehl** [comment]
Thanks, @fazer1929 for the contribution!

**Pull request merged**